dev-python/pypubsub/pypubsub-3.4.2.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
```

Comment:
We are disallowed to add any python2-only new packages to the tree. Python team and treecleaners are doing their best getting rid of python2 altogether. So this ebuild can go...